### PR TITLE
fix: more accurate weights for staking heavy operations

### PR DESCRIPTION
**Pull request by ntn-x2**
Fixes https://github.com/KILTprotocol/ticket/issues/3564.

## How to test

Cd into `chopsticks`, then `yarn install` and `yarn spawn`. Try to call `execute_scheduled_reward_change` with a regular account and `set_inflation` with sudo.

### CODE REVIEW

File: chopsticks/config.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+endpoint:
+    # We fetch relevant storage elements from Spiritnet even tho we are running Peregrine (with sudo).
+    - wss://spiritnet.kilt.io
+db: db/peregrine.sqlite
+runtime-log-level: 5
+port: 50002
+wasm-override: /home/antonio/Developer/kilt-node/target/debug/wbuild/peregrine-runtime/peregrine_runtime.wasm
```
Comment (Ad96el) [inline chopsticks/config.yaml:7]: Can we maybe use the relative path?

Comment (Ad96el) [reply chopsticks/config.yaml:7]: Or is this file only for testing, and we delete it after?

File: chopsticks/config.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import-storage:
+    System:
+        Account:
+            - - - 4seWojfEHrk5YKPahdErazQ3CWEHZYi6NV4gKz5AaejWbRPJ
```
Comment (Ad96el) [inline chopsticks/config.yaml:12]: Is this Alice?

**Review by Ad96el — COMMENTED** [review]
LGTM, one minor comment for chopsticks

**Comment by Ad96el** [comment]
I tested the `execute_scheduled_reward_change` in the chopsticks env

File: chopsticks/config.yaml (hunk 1 above)
Comment (ntn-x2) [reply chopsticks/config.yaml:7]: Yes, I will delete these once you are able to verify the correctness of the solution.

File: chopsticks/config.yaml (hunk 2 above)
Comment (ntn-x2) [reply chopsticks/config.yaml:12]: Was it? I think it's Alice but with Ed25519.

**Review by Ad96el — APPROVED** [review]
LGTM

**Pull request merged**